README.md

```diff
@@ -111,6 +111,13 @@ class Application extends React.Component {
 }
 ```
 
+
```

Comment:
This is the wrong place for this info. We don't need to specify how we've used the callback in our sites, we just need to document the usage of the displayCallback prop here: https://github.com/dennisinteractive/react-gpt/blob/master/docs/api/ReactGPT.md#Bling

We don't need to specify how we've used it in our sites as you've done here

Reply:
of course - how silly, they do proper docs. Too long in Dennis

Reply:
DELETE THIS BIT THEN ILIAS OMG

Reply:
WE ONLY NEED THE DOCS IN ReactGPT.md

Reply:
OMG

Reply:
OMG ILIAS

Reply:
OMFG

Reply:
haha okay done

Reply:
FOMFGFFFF